oasis.md

```diff
@@ -0,0 +1,67 @@
+External OASIS Repositories
+===========================
+
+We offer hosting of non-OSG CVMFS repositories on OASIS. This means that requests to create, rename, remove, or blanking OASIS repositories will come in as GOC tickets. This document contains instructions for handling those tickets.
+
+!!! note "Also see"
+    * [Policy for OSG Mirroring of External CVMFS repositories](https://opensciencegrid.github.io/technology/policy/external-oasis-repos/)
+    * [External OASIS repository](https://opensciencegrid.github.io/docs/data/external-oasis-repos/)
+
+Hosting a repository on OASIS
```

Comment:
I think it would be more helpful to describe or name the sections as ticket responses, for example "Handling a request to host a repository on OASIS", and then have a short intro like "Do the following steps".